pkg/service/model.go

```diff
@@ -0,0 +1,17 @@
+package service
```

Comment:
I think we are safe to move these to the top of `convert.go`.
It's common in Java to create a separate model for the objects, but structs usually live in the same file where are used.
In fact, we may want to add behaviours to the struct later, by using receiver functions, but for now we don't need to worry about that 👍